tests/test_pkce.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2016 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import mock
+import unittest2
+
+from oauth2client import _pkce
+
+
+class PKCETests(unittest2.TestCase):
+
+    @mock.patch('oauth2client._pkce.os.urandom')
+    def test_verifier(self, fake_urandom):
+        rand = (b'\x98\x10D7\xf3\xb7\xaa\xfc\xdd\xd3M\xe2'
```

Comment:
Pull this out to module-scope and structure it as
```
<name> = (
    <bytes literal>
    <bytes literal>
    <bytes literal>
)
```
so that `<name>` can be much helpfully longer than four characters without forcing the literal data all the way to the right side of the text.